frontend/app/forms/application-for-board-of-pardon-e/application-for-board-of-pardon-e.pdf.component.js

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import RenderPage from "../render-page.component.js";
+import { Scoped, a, m } from "kremling";
+import { getClassname } from "../pdf-rendering/data-key.helpers.js";
+import PositionedCheckmark from "../pdf-rendering/positioned-checkmark.component.js";
+import PositionedString from "../pdf-rendering/positioned-string.component.js";
+import moment from "moment";
+
+export default function ApplicationForBoardOfPardonExpungement_Pdf({
+  renderData,
+  data
+}) {
+  return (
+    <React.Fragment>
+      <RenderPage url="/static/forms/application-for-board-of-pardon-e/BOP-Exp-App-08-09-2019-Mail-In-1.png">
+        <Scoped css={css}>
```

Comment:
Direct children of the Scoped component must be dom elements, not React components, in order for the css to be applied. See related https://kremling.js.org/api/scoped-component.html#gotchas

Additionally, it's recommended to use the `useCss` hook for function components instead of Scoped component. See related https://kremling.js.org/api/use-css.html.

You might just have been pattern matching here and copying other components, but let's fix that pattern.

Because of this, I don't think your css is being applied at all, or at least not to everything. Easiest way to fix would be the following:

```js
function Foo() {
  const scope = useCss(css)

  return (
    <RenderPage>
      <div {...scope}>
        <PositionedString />
      </div>
    </RenderPage>
  )
}
```